runtime/simulation/Makefile

```diff
@@ -0,0 +1,91 @@
+ifeq ($(MAKECMDGOALS),)
+  $(shell pkill -f -u $(USER) questa)
+endif
+TOPLEVEL_LANG ?= verilog
+
+SIM = questa
+MODELSIM=$(PWD)/simulate_testbench.sim/sim_1/behav/questa/modelsim.ini
+
+SIM_LIB  = $(PWD)/compile_simlib/questa
+
+SCRIPT_FILE = $(PWD)/simulate_testbench.sim/sim_1/behav/questa/pe_compile.do; \
+do $(PWD)/simulate_testbench.sim/sim_1/behav/questa/pe_elaborate.do; \
+exec cp $(PWD)/simulate_testbench.sim/sim_1/behav/questa/system_tapasco_status_base_0.mif .;
+
+SIM_IP := composition.zip
+BOARD_PART := xczu3eg-sbva484-1-e
+IP_DEFS := esa.informatik.tu-darmstadt.de:sim:*
+SIM_PORT := 4040
+export SIM_PORT
+
+RUN_ARGS=-noautoldlibpath
+GUI:=0
+
+RTL_LIBRARY = xil_defaultlib
+TOPLEVEL := pe
+MODULE   := simulation
+PYTHON_PATH := .
+PYGPI_ENTRY_POINT := simulation:sim_entry
+
+SHELL=/bin/bash
+
+ifeq ("$(wildcard .venv)", "")
+  CONFIGURE_NEEDED = 1
+else
+  SIM_ARGS=-lib xil_defaultlib pe_opt -suppress 7061 -suppress 12003 -onfinish exit -pli "$(shell cocotb-config --prefix)/cocotb/libs/libcocotbvpi_modelsim.so"; \#
+  include $(shell cocotb-config --makefiles)/Makefile.sim
+endif
+
+ifeq ($(TAPASCO_HOME_RUNTIME),)
+  $(error "Tapasco Home Runtime not found. Please source correct tapasco-setup.sh!")
+endif
+
+
+configure:
+	python -m venv .venv
+	source .venv/bin/activate; \
+	pip install -r requirements.txt
+
+test: sim
+	python3 combine_results.py
+
+simlib_questa:
+	echo "compile_simlib -simulator questa -simulator_exec_path {/opt/cad/mentor/questa/questa-2021.3/questasim/bin} -gcc_exec_path {/opt/cad/mentor/questa/questa-2021.3/questasim/gcc-7.4.0-linux_x86_64/bin} -family zynquplus -language all -library all -dir {compile_simlib/questa}" > questa.tcl
```

Comment:
Please remove the absolute paths. AFAIK Vivado requires it, so maybe retrieve it through `which`?